cmd/mockery.go

```diff
@@ -145,9 +143,33 @@ func GetRootAppFromViper(v *viper.Viper) (*RootApp, error) {
 	return r, nil
 }
 
+func getFilters(names []string, log zerolog.Logger) ([]*regexp.Regexp, bool) {
+
+	const regexMetadataChars = "\\.+*?()|[]{}^$"
+
+	var hasRegex bool
+	var filters []*regexp.Regexp
+
+	for _, name := range names {
+		if strings.ContainsAny(name, regexMetadataChars) {
+			hasRegex = true
+			filter, err := regexp.Compile(name)
+			if err != nil {
+				log.Fatal().Err(err).Msgf("Invalid regular expression provided to -name")
```

Comment:
The error should be returned and bubbled up the stack. I generally avoid `log.Fatal()` as it's non-graceful.